exams/models.py

```diff
@@ -30,8 +36,65 @@ class ExamProfile(Model):
     status = models.CharField(
         max_length=30,
         null=False,
-        choices=PROFILE_STATUS_CHOICES
+        choices=PROFILE_STATUS_CHOICES,
+        default=PROFILE_PENDING,
     )
 
     def __str__(self):
         return 'Exam Profile "{0}" with status "{1}"'.format(self.id, self.status)
+
+
+class ExamAuthorization(TimestampedModel):
+    """
+    Tracks state of an exam authorization
+
+    Exam Authorization record are sent to Pearson authorizing an individual for a specific exam.
+    This model exists so we can track state on our side and corresponds to a row in a Pearson EAD file.
+    """
+    OPERATION_ADD = 'add'
+    OPERATION_DELETE = 'delete'
+    OPERATION_UPDATE = 'update'
+
+    OPERATION_CHOICES = tuple((op, op.capitalize()) for op in [
```

Comment:
I am pretty sure you do not need to use `tuple` explicitly, but feel free to leave as is